backend/classify.functions.php

```diff
@@ -30,16 +31,19 @@ function ipCheck($hostname, $debug = false) {
         }
         $reader = new Reader(__DIR__.'/GeoLite2-City.mmdb');
         $record = $reader->city($hostname);
-        if ( $debug > 1 ) {
+        if ( $debug ) {
             var_dump($record);
+            echo "\n";
         }
         $continent = $record->continent->code;
         $country   = $record->country->isoCode;
         $longitude  = $record->location->longitude;
 
     } catch (\Exception $e) {
         if ( $debug ) {
-            echo '<!-- '.$e->getMessage().' -->'."\n";        
+            log_echo('<!-- '.$e->getMessage().' -->'."\n");
+        } else {
+            error_log($e->getMessage());
```

Comment:
I feel these should be flipped. If it's in debug, it should output the message, but not send it to sentry because someone is already debugging it. If it's in production (or not debug mode) it should send the message to sentry (although not echo the error out loud?).